docker-compose.yml

```diff
@@ -1,6 +1,20 @@
 version: '3.8'
 
 services:
+  app:
+    build:
+      context: .
+      dockerfile: ./.docker/app/Dockerfile
+    networks:
+      - mariadb
+    volumes:
+      - .:/app
+    environment:
+      - ENVIRONMENT=DEVELOPMENT
+    ports:
+      - "127.0.0.1:80:80"
```

Comment:
Why specify explicitly binding to `127.0.0.1` when `80:80` would be enough?

Reply:
Good practice, eg. for anyone using podman instead of docker.